HVSample/src/main/java/com/microsoft/hsg/android/hvsample/MainActivity.java

```diff
@@ -117,126 +130,132 @@ public void onConnected() {
 
 	public void onError(Exception e) {
 		Toast.makeText(
-                MainActivity.this, 
-                e.getMessage(), 
-                Toast.LENGTH_LONG).show();
+				MainActivity.this,
+				e.getMessage(),
+				Toast.LENGTH_LONG).show();
 	}
 
 	@Override
 	public boolean onOptionsItemSelected(MenuItem item) {
-	    switch (item.getItemId()) {
-	        case R.id.action_connect:
```

Comment:
I checked on this and it seems ok to me

---
In reply to: [117543086](https://github.com/Microsoft/healthvault-java-sdk/pull/5#discussion_r117543086) [](ancestors = 117543086)